module4/gift-certificate-system/core/src/main/java/com/epam/esm/model/Role.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.esm.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Set;
+@Data
+@AllArgsConstructor()
+@NoArgsConstructor
```

Comment:
Why do you need so many constructors? @Data annotation includes @RequiredArgsConstructor that should add default constructor for this class. Also I can not see where do you use @AllArgsConstructor in code.